modules/oe_content_publication/config/install/field.field.node.oe_publication.oe_publication_type.yml

```diff
@@ -10,7 +10,7 @@ id: node.oe_publication.oe_publication_type
 field_name: oe_publication_type
 entity_type: node
 bundle: oe_publication
-label: Type
+label: 'Resource type'
```

Comment:
Configuration has to be updated in post_update (00003 - create new fields, 00004 - update existing fields, 00005 - update form view, 00006 - update display view).

Reply:
This is done.